ablator/modules/scheduler.py

```diff
@@ -178,7 +178,7 @@ class PlateuaConfig(SchedulerArgs):
     patience: int = 10
     min_lr: float = 1e-5
     mode: str = "min"
-    factor: float = 0.0 # TODO {fixme} this is error prone -> new_lr = 0
+    factor: float = 0.1
     threshold: float = 1e-4
     verbose: bool = False
     # TODO fix mypy errors for custom types
```

Comment:
is this addressed?

Reply:
Yes, all custom types should be fixed by plugin, but here it is ty.Literal from python typing.

Reply:
@h21565897 it should throw an error if inheriting from `ty.Literal` can you please investigate.